packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -138,6 +138,30 @@ export const taggingNameCapitalSetting = {get: () => startCase(taggingNameSettin
 export const taggingNamePluralSetting = {get: () => pluralize(taggingNameSetting.get())}
 export const taggingNamePluralCapitalSetting = {get: () => pluralize(startCase(taggingNameSetting.get()))}
 export const taggingNameIsSet = {get: () => taggingNameSetting.get() !== 'tag'}
+export const usePluralTagNameSetting = {get: () => !isLWorAF && taggingNameIsSet.get()};
```

Comment:
The name of this shouldn't start with `use` (since that would imply it's a React hook)